website/www/site/content/en/contribute/release-guide.md

```diff
@@ -253,10 +253,19 @@ Attention: Only committer has permission to create release branch in apache/beam
 
 Release candidates are built from a release branch.
 As a final step in preparation for the release, you should create the release branch, push it to the Apache code repository, and update version information on the original branch.
+The final state of the repository should match this diagram:
 
-There are 2 ways to cut a release branch: either running automation script(recommended), or running all commands manually.
+<img src="/images/cut-release-branch.png" alt="Increment minor version on master branch and set Dataflow container version on release branch" width="100%">
 
-After following one of these processes you should manually update `CHANGES.md` on `master` by adding a new section for the next release.
+- The `master` branch has the SNAPSHOT/dev version incremented.
```

Comment:
I assume script already handles this information is just to double check ? (probably good to explicitly mention that for clarity).